neuralmonkey/decoders/sequence_labeler.py

```diff
@@ -139,3 +140,59 @@ def feed_dict(self, dataset: Dataset, train: bool = False) -> FeedDict:
             fd[self.train_weights] = paddings.T
 
         return fd
+
+
+class CRFLabeler(SequenceLabeler):
+    """CRF Classifier assing a label to each encoder's state."""
+
+    def __init__(self,
+                 name: str,
+                 encoder: Union[RecurrentEncoder, SentenceEncoder],
+                 vocabulary: Vocabulary,
+                 data_id: str,
+                 dropout_keep_prob: float = 1.0,
+                 save_checkpoint: Optional[str] = None,
+                 load_checkpoint: Optional[str] = None) -> None:
+        SequenceLabeler.__init__(self, name, encoder, vocabulary, data_id,
+                                 dropout_keep_prob, save_checkpoint,
+                                 load_checkpoint)
+
+        self.transition_params = tf.get_variable(
```

Comment:
nepoužíváš scope tohohle model partu: obalit do ``with self.use_scope():`` nebo něco takovýho. Je to funkce na model partu